recipes/ostaphep/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+
+mkdir -p build-dir
+cd build-dir
+
+
+cmake -LAH \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DCMAKE_INSTALL_NAME_DIR="${PREFIX}/lib"\
+    ../ostap
+
+make -j${CPU_COUNT}
+make install
+
+
+
+
+
+ln -s "${PREFIX}/ostap" "${SP_DIR}/"
```

Comment:
Can the CMake configuration be patched to install into the site-packages directory instead of symlinking? I think you can use the `Python_SITELIB` variable that is set by `FindPython`.

A less nice alternative but still valid option is to add an option to the CMake config.